agent/src/index.test.ts

```diff
@@ -647,10 +647,14 @@ describe('Agent', () => {
             }
         })
 
+        if (process.env.CODY_NODE_VERSION === '16') {
+            it('Skipping edit chat test in Node.js v16 because of unsupported API used', () => {})
+            return
+        }
+
         it(
```

Comment:
FYI You can import this helper here and use it like this https://sourcegraph.com/github.com/sourcegraph/cody@8968a9b51794e046781c7c98bc8aff337e4fb429/-/blob/agent/src/cli/evaluate-autocomplete/AutocompleteMatcher.test.ts?L18%3A10-18%3A18
```suggestion
        it.skipIf(isNode16())(
```